From 00848de37701b5496e663daf0ab22633b3ab90fb Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 04 Oct 2018 14:00:29 +0300
Subject: [PATCH] Makefile: finally remove TODO

The TODO has already been done, but forgotten and not deleted.
Removing now.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 51f9fa2..857e5e6 100644
--- a/Makefile
+++ b/Makefile
@@ -67,7 +67,6 @@
 	$(error HL_RELEASE_VER is undefined, usually looks like x.y.z)
 endif
 
-#TODO: add btl once it is separated from br-ext
 RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot fw_loader btl
 RELEASE_TAG_POINTER := latest-embedded-release
 
